contrib/containers/guix/scripts/guix-check

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+cd /src/dash
+COMMIT_ID=guix-build-$(git rev-parse --short=12 HEAD)
+
+printf "Binaries:\n"
+( \
+SRC_PATH_PREFIX=guix-build-${COMMIT_ID}/distsrc- && \
+sha256sum ${SRC_PATH_PREFIX}*/src/dash{d,-cli,-tx,-wallet}{,.exe} && \
+sha256sum ${SRC_PATH_PREFIX}*/src/qt/dash-qt{,.exe} && \
+sha256sum ${SRC_PATH_PREFIX}*/src/test/test_dash{,.exe} \
+) | sort -k 2
```

Comment:
That script exists to mirror the CI's capabilities in interactive mode. I originally planned to have CI call the script through the container but since the checksum calculation is done in a separate "step", we have to re-start the container just to invoke `guix-check` which felt wasteful.

Though we could just directly call the script like `./contrib/containers/guix/scripts/guix-check`. Thoughts?

Reply:
Resolved with https://github.com/dashpay/dash/pull/5464